lib/nagios_dnssec.rb

```diff
@@ -88,11 +88,11 @@ def process_log(options, output, syslog = nil)
   end
   case options.nagios_verbosity
   when 0..1
-    nagios_buffer = [worst_nagios_output].to_s.chomp
```

Comment:
I would change this first line to

  nagios_buffer = [worst_nagios_output.to_s.chomp]

to avoid an extra line feed (in case worst_nagios_output already contains one) but otherwise I second this patch.

Can it be merged please? I just spent unnecessary time fixing this bug myself before finding the existing pull request.

Thanks

/Fredrik